src/main/java/core/basesyntax/SalaryInfo.java

```
@@ -1,7 +1,88 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    public static final String DATE_FORMAT = "dd.MM.yyyy";
+
+    private LocalDate dateFrom;
+    private LocalDate dateTo;
+    private String[] names;
+    private int[] salarySum;
+    private final DateTimeFormatter dateFormatter =
+            DateTimeFormatter.ofPattern(DATE_FORMAT);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        if (names == null || data == null) {
+            System.out.println("Error! Input parameters are wrong");
+            return "";
```

It is better to throw an exception here